File: app/src/main/java/com/rafamatias/nlp/data/entity/SentimentResponse.java
package com.rafamatias.nlp.data.entity;

public class SentimentResponse {
    private Result documentSentiment;

    public double getScore() {
        if(documentSentiment == null) {
            return 0.0;
        }

        return documentSentiment.score;
    }


    static class Result {
        private double score;
    }
}

File: data/src/main/java/com/shrpereira/tweetsense/data/remote/sentiment/SentimentResponse.kt
package com.shrpereira.tweetsense.data.remote.sentiment

import com.google.gson.annotations.SerializedName

data class SentimentResponse(
	@SerializedName("documentSentiment") val documentSentiment: DocumentResponse
)

data class DocumentResponse(
	@SerializedName("score") val score: Double?
)